manifests/n/NoMachine/NoMachine/8.5.3_1/NoMachine.NoMachine.installer.yaml

```
@@ -0,0 +1,21 @@
+# Created using wingetcreate 1.2.6.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: NoMachine.NoMachine
+PackageVersion: 8.5.3_1

```

```suggestion
PackageVersion: 8.5.3
```
Must match App&Features / Control Panel

---

Fixed.